packages/maumau-server/src/game/game-state.ts

```diff
@@ -111,15 +112,15 @@ export default class GameState {
     }
 
     for (const player of this.state.players) {
-      if (player.isDisconnected()) {
+      if (player.bot == undefined && player.isDisconnected()) {
```

Comment:
```suggestion
      if (Boolean(player.bot)) {
          continue;
      }
      if (player.isDisconnected()) {
      ....
      }
```
nit: Ist vllt besser zu lesen mit einem `continue` oben